src/main/java/seedu/address/logic/Messages.java

```diff
@@ -45,10 +50,9 @@ public static String getErrorMessageForDuplicatePrefixes(Prefix... duplicatePref
     /**
      * Formats the {@code person} for display to the user.
      * <p>
-     * The updatedAt field is not included due to its nondeterministic nature.
-     * It is impossible to predict the value of the updatedAt field.
-     * This ensures that unit testing for {@link EditCommand}
-     * is not affected.
+     * The {@code updatedAt} field is not included due to its nondeterministic nature.
+     * It is impossible to predict the value of the {@code updatedAt} field.
+     * This ensures that unit testing for {@link EditCommand} is not affected.
```

Comment:
Missing `</p>` tag here for javadoc

Reply:
The closing `</p>` tag can be [omitted](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/p#technical_summary:~:text=Phrasing%20content.-,Tag%20omission,-The%20start%20tag).